minutes/2022-06-13.md

```diff
@@ -0,0 +1,61 @@
+# Meeting Minutes for February 25, 2021
+
+## Attendees
+
+@fviernau, @mbochtler, @mnonnenmacher, @sschuberth, @tsteenbe
```

Comment:
Marcel GitHub handle is [MarcelBochtler](https://github.com/MarcelBochtler) not @mbochtler